extractor/oidc.go

```diff
@@ -119,5 +123,7 @@ func (o *oidcExtractor) Process(ctx context.Context, cfg *oauth2.Config, code st
 		return nil, errors.New("Invalid email domain, expecting " + o.emailDomain)
 	}
 
+	o.log.Error("UserInfo", zap.Any("userinfo", params))
```

Comment:
This does not seem to be an error condition, but it is being logged as an error. Should it be a debug log line instead?